JSQMessagesViewController/Controllers/JSQMessagesViewController.m

```diff
@@ -63,6 +63,8 @@ @interface JSQMessagesViewController () <JSQMessagesInputToolbarDelegate,
 
 @property (weak, nonatomic) UIGestureRecognizer *currentInteractivePopGestureRecognizer;
 
+@property (assign, nonatomic) BOOL interactivePopGestureFirstResponderState;
```

Comment:
this needs a better name. how about `textViewWasFirstResponderDuringInteractivePop` ?